build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/MrjarPlugin.java

```diff
@@ -154,7 +156,6 @@ private void addJar(Project project, SourceSet sourceSet, int javaVersion) {
         project.getConfigurations().register("java" + javaVersion);
         TaskProvider<Jar> jarTask = project.getTasks().register("java" + javaVersion + "Jar", Jar.class, task -> {
             task.from(sourceSet.getOutput());
-            task.getArchiveClassifier().set("java" + javaVersion);
```

Comment:
You're accidentally removing the fix we added for the duplicated jar

Reply:
Wow, thanks for identifying this Rene!